src/lib/components/eventModal.svelte

```diff
@@ -279,7 +282,7 @@
         </div>
     {:else}
         <div class="input-text-wrapper" style="--amount-of-buttons:2">
-            <div type="text" readonly style="min-height: 2.5rem;">
+            <div {...attributes} style="min-height: 2.5rem;">
```

Comment:
I don't think we need those attributes in the first place right?

Reply:
![image](https://user-images.githubusercontent.com/45005369/216934004-e2d42ccb-c797-4f79-b20b-1f904acd2e73.png)
We do, we use them for styling :thinking: